include/verilated_funcs.h

```diff
@@ -819,6 +819,20 @@ static inline IData VL_CLOG2_W(int words, WDataInP const lwp) VL_PURE {
     }
     return 0;
 }
+static inline uint32_t VL_CTZ_I(IData val) {
+    if (VL_UNLIKELY(!val)) return 0;
+#if defined(__GNUC__) && !defined(VL_NO_BUILTINS)
+    return __builtin_ctz(val);
+#else
+    int bit = 0;
+    val = ~val;
+    while (val & 1) {
+        ++bit;
+        val >>= 1;
+    }
+    return bit;
```

Comment:
Can do much better, see https://www.oreilly.com/library/view/hackers-delight/0201914654/0201914654_ch05lev1sec4.html

However given there's only 3 bits in the value, I suspect hardcoding vs using CTZ may be better.

Reply:
I agree. However, to simplify this for the case when the number is 8-bit integer and only one bit can be set at the time I will use `VL_CLOG2_I`. But if you want I may implement this using constexpr lookup table with those 8 cases.

The code for `VL_CTZ_I` I took from `V3Const`, thus I think it will be beneficial to improve the algorithm there.

Reply:
>The code for VL_CTZ_I I took from V3Const, thus I think it will be beneficial to improve the algorithm there.

Note for future code in V3Const is slow path, run only at compile time, so is readable instead of performance-optimized.